plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8661,6 +8662,19 @@ protected TestTable createTableWithDefaultColumns()
         throw new SkipException("Hive connector does not support column default values");
     }
 
+    @Override
+    protected OptionalInt maxTableNameLength()
+    {
+        // This value depends on HMS type
+        return OptionalInt.of(255);
+    }
+
+    @Override
+    protected void verifyTableNameLengthFailurePermissible(Throwable e)
+    {
+        assertThat(e).hasMessageContaining("Failed to create directory");
```

Comment:
Is it same on Linux CI and Mac OS development box?

Reply:
Yes, it's same.